### PR TITLE
dev-libs/amdgpu-pro-opencl: version bumped to 17.40.492261.

### PR DESCRIPTION
**Pull request by brovko**
Bug: https://bugs.gentoo.org/635236

Added miners support: "Enabling Large Page Support" kernel parameter: amdgpu.vm_fragment_size=9

Package-Manager: Portage-2.3.12, Repoman-2.3.3

### CODE REVIEW

**Comment by brovko** [comment]
Works well on my home computer several days (24h mining).

**Comment by mkszuba** [comment]
Thank you for your contribution! I'll run some tests of the new version on my AMD box some time during the next few days so you can expect it to be merged by the end of the week.

In the meantime, it would be great if you could split your commit into two - one bumping the version and one adding you as maintainer. In Gentoo we follow the "one logical change per commit" policy and these are two different things.

By the way, since in the end AMD have merged large-page support into the general-purpose driver and this is a version bump rather than having a new package added to Gentoo, technically you needn't become a maintainer. That said, you are very much welcome if you want to become a co-maintainer of this package! It is always helpful to have a second set of eyes on things and doubly so for packages pertaining to hardware. What do you think?

File: dev-libs/amdgpu-pro-opencl/amdgpu-pro-opencl-17.40.492261.ebuild [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+src_prepare() {
+	default
+
+	cd "${S}/opt/${SUPER_PN}/lib/x86_64-linux-gnu" || die
+	patchelf --set-rpath '$ORIGIN' libamdocl64.so || die "Failed to fix library rpath"
+}
+
+src_install() {
+
```
Comment (SoapGentoo) [inline dev-libs/amdgpu-pro-opencl/amdgpu-pro-opencl-17.40.492261.ebuild]: remove spurious empty line

**Comment by brovko** [comment]
Updated pull request (removed empty line). Split commit to to parts: version bump and co-maintainer.

**Comment by mkszuba** [comment]
Merged! I have, however, skipped the messages about enabling large-page support because having read about how it is enabled, I discovered it is out of scope of this package - vm_fragment_size is an option of the amdgpu kernel module, not of AMD DRM or OpenCL libraries. It is mentioned in AMDGPU-Pro release notes because AMD ships their own copy of that module.

For the record, vanilla Linux kernels support the setting of amdgpu.vm_fragment_size (with the default remaining 64 kB) since 4.14. Moreover, starting with 4.15 (the relevant patch was submitted on the 18th of September and has since been merged) the module will use 2-megabyte fragments by default for Sea Islands and newer GPUs.

**Pull request closed without merging**